layers/+lang/mercury-layer/packages.el

```diff
@@ -0,0 +1,77 @@
+;;; packages.el --- mercury layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2018 Sylvain Benner & Contributors
+;;
+;; Author: Ludvig Böklin <ludvig.boklin@protonmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+;; See the Spacemacs documentation and FAQs for instructions on how to implement
+;; a new layer:
+;;
+;;   SPC h SPC layers RET
+;;
+;;
+;; Briefly, each package to be installed or configured by this layer should be
+;; added to `mercury-packages'. Then, for each package PACKAGE:
+;;
+;; - If PACKAGE is not referenced by any other Spacemacs layer, define a
+;;   function `mercury/init-PACKAGE' to load and initialize the package.
+
+;; - Otherwise, PACKAGE is already referenced by another Spacemacs layer, so
+;;   define the functions `mercury/pre-init-PACKAGE' and/or
+;;   `mercury/post-init-PACKAGE' to customize the package as it is loaded.
+
+;;; Code:
```

Comment:
From here the your code begins please remove all comments between line 39 and line 12